cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -66,24 +66,27 @@ def wrapper(*args, **kwargs):
 SEND_CATALOG_INFO_SIGNAL = SettingToggle('SEND_CATALOG_INFO_SIGNAL', default=False, module_name=__name__)
 
 
-def create_catalog_data_for_signal(course_key: CourseKey) -> Optional[CourseCatalogData]:
+def create_catalog_data_for_signal(course_key: CourseKey) -> (Optional[datetime], Optional[CourseCatalogData]):
```

Comment:
One nit, although this code was already here. I really like using prefix of `_` for private-ish methods so I know I can be less concerned about refactors.

Reply:
[fixed]